src/analyze/analyze.c

```diff
@@ -1051,7 +1051,7 @@ static int analyze_critical_chain(int argc, char *argv[], void *userdata) {
         }
         unit_times_hashmap = h;
 
-        (void) pager_open(arg_pager_flags);
```

Comment:
Isn't the chunk changing the `pager_open()` return type missing?

Reply:
Arggh, git hard.